### PR TITLE
Add Clear Tasks function

### PR DESCRIPTION
**Pull request by Arch-Noize**
I added a new function to my To-Do list project:
- Whenever a checkbox is changed, the `completed` status of the items of my To-Do list changes too!
- A button clears all items that have already been marked completed.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
-/* eslint-disable no-alert, no-plusplus, */
+/* eslint-disable no-alert, no-plusplus */
```
Comment (cynthiainga) [inline src/index.js]: - [ ] Please kindly do not disable linters, fixing linters help us to write clean, clear, and strong codebase 🙃 😉

File: src/modules/completed.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-plusplus */
```
Comment (cynthiainga) [inline src/modules/completed.js]: - [ ] Please kindly do not disable linters, fixing linters help us to write clean, clear, and strong codebase 🙃 😉

File: src/modules/edit.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
-/* eslint-disable no-alert, no-plusplus, */
+/* eslint-disable no-plusplus, import/no-mutable-exports */
```
Comment (cynthiainga) [inline src/modules/edit.js]: - [ ] Please kindly do not disable linters, fixing linters help us to write clean, clear, and strong codebase 🙃 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Arch-Noize, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] The clear completed tasks button work correctly ✔️ 
- [ ] The status of the task can be updated to true or false ✔️ 
- [ ] Correct GitHub flow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/edit.js (hunk 3 above)
Comment (Arch-Noize) [reply src/modules/edit.js]: Hi, thank you for your review. 

Part of the documentation on ESlint and the material given by Microverse indicated that some of the options given by ESLint are meant to be disabled here and there. Such as max-number-classes. Previous reviewers didn't tag me on this. 

Is there a way to better follow the guidelines provided by such activities? As to what should and shouldn't be disabled?

File: README.md
Comment (shahadat3669) [inline README.md:1]: - [ ] [Optional] 👍 Your README file looks great! However, I would recommend adding the current technology stack to the Tech Stack section 🛠️📚. This addition will enhance your README and provide readers with a clearer understanding of the project. Keep up the good work! 💪

image

**Review by shahadat3669 — APPROVED** [review]
Hi @Arch-Noize , 

Your project is complete! As the previous reviewer mentioned you fixed all the linters. Keep it up. 👍 it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ Done everything as the previous reviewer requested. 👍 
✅ No linter errors
✅ Good PR Title and Description



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

## Thank you for your time 🙏 and call me if you need help or clarification ☎️
| GitHub | LinkedIn |  Slack |  E-Mail    |
| --- | --- | --- | --- |
|   [ GitHub ](https://github.com/shahadat3669)   |   [ LinkedIn ](https://www.linkedin.com/in/shahadat-cseng/)    |  [ Slack ](https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V)   |  [ Gmail ](mailto:shahadat3adsf669@gmail.com?subject=Microverse%20Code%20Review)|


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**